File: app/src/main/java/com/wilinz/yuetingmusic/data/services/user/MusicInfoService.java
package com.wilinz.yuetingmusic.data.services.user;

import com.wilinz.yuetingmusic.data.model.MusicUrl;

import io.reactivex.rxjava3.core.Observable;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface MusicInfoService {
    @GET("/song/url")
    Observable<MusicUrl> getMusicUrls(@Query("id") String idList);
}

File: app/src/main/java/com/wilinz/yuetingmusic/data/services/user/MusicInfoService.kt
package com.wilinz.yuetingmusic.data.services.user

import com.wilinz.yuetingmusic.data.model.MusicUrl
import io.reactivex.rxjava3.core.Observable
import retrofit2.http.GET
import retrofit2.http.Query

interface MusicInfoService {
    @GET("/song/url")
    fun getMusicUrls(@Query("id") idList: String?): Observable<MusicUrl>
}